src/main/kotlin/org/malachite/estella/services/UserService.kt

```diff
@@ -52,5 +53,10 @@ class UserService(
 
     fun getUserByEmail(email: String): User? = userRepository.findByMail(email).orElse(null)
 
+    fun registerUser(user:User):User {
+        val user = addUser(user)
+        mailService.sendMail(mailService.userRegistrationMailPayload(user))
```

Comment:
Wydaje mi się, że jest już dużo za dużo tych templatek w Mail Service. Weź je wszystkie wydziel do osobnego obiektu / osobnych funkcji i zapisz je w osobnym pliku - będzie się do nich łatwiej dostać, nie odwołując się bezpośrednio do mailService

Reply:
Dlatego wydawało mi się że te dwa taski trzeba robić w jednym PR'rze, bo teraz muszę zmergować zmiany na drugim, żeby przenieść template też tam XD

Reply:
A co Cię w tym powstrzymuje?

Reply:
A tak w ogóle mam wydzielić teksty, czy też payloady, bo jak payloady to w serwisie nic nie zostanie XD, oprócz wysyłania maili XD

Reply:
> A co Cię w tym powstrzymuje?

Jest to dodatkowa robota plus mimo wszystko lepiej mergować na drugim PR jak ten zostanie już zmergowany